problems/removeNumbersAtOddIndices.js

```diff
@@ -1,9 +1,13 @@
+const removeOddNumbers = require("./removeOddNumbers");
+
 /** Remove all numbers at odd indices from an array
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+function removeNumbersAtOddIndices(arr) {
+    let ar = ['zero', 'one', 'two', 'three'];
+    
+    return arr.shift
```

Comment:
shift() removes the first item of an array. 
We want to remove odd indicies. Look into how you might get this to work with filter.